docs/deployments/dsk-agent/ads/docker-compose.override.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.3"
+
+services:
+  ads-source:
+    image: ${DOCKER_REGISTRY-tributech.azurecr.io/}ads-source:${TAG:-latest}
```

Comment:
should we tag a specific version here? as now we have versioned docs (otherwise when we create a new version the old version would still display lastest)

Reply:
We are using an Environment File as Standard Deployment which defines always the latest Version. latest is here really just a Fallback Value and should never been hit. 
I had an Conversation with Chris though and we will change the Tags for the Value Sources to SOURCE_TAG and the Agent will get a AGENT_TAG. Before it was all declared as Tag but we everyday we strife further away from a unified Versioning.